bridge/elrond/client.go

```diff
@@ -265,13 +278,20 @@ func (c *client) GetActionIdForProposeTransfer(_ context.Context, batch *bridge.
 }
 
 // WasProposedSetStatus returns true if the proposed set status was triggered
-func (c *client) WasProposedSetStatus(_ context.Context, batch *bridge.Batch) bool {
+func (c *client) WasProposedSetStatus(ctx context.Context, batch *bridge.Batch) bool {
 	valueRequest := newValueBuilder(c.bridgeAddress, c.address.AddressAsBech32String(), c.log).
 		Func("wasSetCurrentTransactionBatchStatusActionProposed").
 		BatchId(batch.Id)
 
-	for _, tx := range batch.Transactions {
-		valueRequest = valueRequest.BigInt(big.NewInt(int64(tx.Status)))
+	newBatch, err := c.GetPending(ctx)
+	if err != nil {
+		c.log.Error("Elrond: get pending batch failed in WasProposedSetStatus", "error", err.Error())
```

Comment:
optional: `err` instead of `err.Error()`

Reply:
done